weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -651,10 +651,24 @@ function makeCurlText(
   const baseUrl = (window as any).CONFIG.TRACE_BACKEND_BASE_URL;
   const filterStr = JSON.stringify(
     {
-      op_names: filter.opVersionRefs,
-      input_refs: filter.inputObjectVersionRefs,
-      output_refs: filter.outputObjectVersionRefs,
-      parent_ids: filter.parentIds,
+      op_names:
+        filter.opVersionRefs && filter.opVersionRefs.length > 0
```

Comment:
Actually maybe define consts:

```typescript
const hasOpVersionRefs = filter.opVersionRefs !== undefined && filter.opVersionRefs.length > 0;
```

to use above and below.

Reply:
... fine